.gitlab-ci.yml

```diff
@@ -7,14 +7,15 @@
 .node_anchors:
   # `only` (also used for `except` where applicable)
   only_branch_master_parent_repo: &only_branch_master_parent_repo
-    - 'master@saltstack-formulas/rabbitmq-formula'
+    - 'master@saltstack-formulas/template-formula'
```

Comment:
isn't this supposed to point to `rabbitmq-formula` repository?

Reply:
I forked latest template-formula and ported rabbitmq-formula to that.  

Issue #40 said PKiman/saltstack_rabbitmq_extended was not working. I introduced `rabbitmq.rabbitmqadmin.config`  to address #40 as best I could - see pillar.example for what is working in CI/CD.

 I can add a flag to make latest optional. 
 
 I'd prefer to extend CI/CD in some other PR. I'm swamped with work at the moment.

thanks for feeback @daks

Reply:
If those from PKiman don't work, forget about this comment :)
I thought that moving them from _states and _modules would be a regression because you can't anymore use them like vanilla execution and states modules (like for example using them directly for `salt` command-line interface)

Reply:
"I can add a flag to make latest optional." could be great in fact, thanks!

Reply:
Thanks - I'm just trying to get CI over the line - Its frustratingly stubborn. 
- Cent 7 plugin state failing with locale error.
- Cent 8 service issue fails (no locale issue)
- conversion job failure - I have no idea what that job is doing.

Yeah, I'm not exactly porting PKiman here, but I was checking his pillar data and states to see what is needed here.  Those extra states are working in my PR (except upstream which I disabled in pillar data).  Hopefully I can get CI working today.